plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -724,4 +739,12 @@ public String getExtensionSource(Configurator c) throws IOException {
         return jar.substring(0, jar.lastIndexOf('.'));
     }
 
+    @Restricted(NoExternalUse.class)
+    public static String printThrowable(@NonNull Throwable t) {
+        String s = Functions.printThrowable(t)
+            .split("at io.jenkins.plugins.casc.ConfigurationAsCode.export")[0]
```

Comment:
perhaps best if it was togglable since there could be some case where it is export or something deeper at fault?

Reply:
Doesn't really make sense, it introduces too much unnecessary complexity in my mind at least.